src/utils/config-helper.ts

```diff
@@ -69,6 +69,10 @@ export function fetchConfig (): Promise<any> {
 
       const ldClientId: string = response.data['LD_CLIENT_ID'];
       (<any>window).ldClientId = ldClientId
-      console.info('Set Launch Darkly Client ID.')
+      console.info('Set LD Client Id.')
```

Comment:
Please use previous text.

Note that this code has evolved a bit since this, but it should be easy to merge it.